<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiImportDialog</class>
 <widget class="QDialog" name="MultiImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="addFilesPushButton">
     <property name="text">
      <string>Add Files...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Dataset Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="text">
        <string>Dataset0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="abscissaLabel">
       <property name="toolTip">
        <string>The title of the spectral x axis.</string>
       </property>
       <property name="text">
        <string>Abscissa Title</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="abscissaLineEdit">
       <property name="text">
        <string>Raman Shift (cm⁻¹)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ordinateLabel">
       <property name="toolTip">
        <string>The label of the spectral y axis.</string>
       </property>
       <property name="text">
        <string>Ordinate Title</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ordinateLineEdit">
       <property name="text">
        <string>Intensity (a.u.)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rowsLayout">
     <item>
      <widget class="QLabel" name="rowsLabel">
       <property name="text">
        <string>Rows</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rowSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="columnsLabel">
       <property name="text">
        <string>Columns</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="colSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="cellsLabel">
       <property name="text">
        <string>Total Cells: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="countLabel">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="filenameTableWidget"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MultiImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MultiImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
